client/rpc/block_converter_test.go

```diff
@@ -0,0 +1,24 @@
+
+func TestConvertBlockResult(t *testing.T) {
+	cdc := codec.New()
+
+	var block ctypes.ResultBlock
+	cdc.MustUnmarshalJSON([]byte(defaultBlockJson), &block)
+
+	convertedBlock := ConvertBlockResult(&block)
+	assert.Equal(t, "cosmosvaloper1az62lz2mxqwr6ugge2f6pwfrfcaz5jephqhvkt", convertedBlock.BlockMeta.Header.ProposerAddress.String())
+
+	convertedBlockString := string(cdc.MustMarshalJSON(&convertedBlock))
+	assert.NotContains(t, "E8B4AF895B301C3D7108CA93A0B9234E3A2A4B21", convertedBlockString)
```

Comment:
I'd like to see a test covering the case with *ctypes.ResultBlock argument being null.

Reply:
Added, thanks for the hint